pkg/storage/ec/client.go

```diff
@@ -104,7 +104,7 @@ func (ec *ecClient) Put(ctx context.Context, nodes []*pb.Node, rs eestream.Redun
 			}
 			ps, err := ec.d.dial(ctx, n)
 			if err != nil {
-				zap.S().Errorf("Failed putting piece %s -> %s to node %s: %v",
+				zap.S().Errorf("Failed dialing for piece %s -> %s to node %s: %v",
```

Comment:
Should we fix it the same way for the Delete method?

Reply:
Yup

Reply:
done